run.py

```diff
@@ -376,6 +363,8 @@ def main():
             raise Exception("Error applying topup to inputs") from e
 
 
+        #TODO: Make this run on the input images, PLUS any corrected images
```

Comment:
RE: main() line 328: context.work_dir creates the directory.

Reply:
True, but I'm leaving it in for redundancy if that's ok.